chevah/empirical
# [#3434] Add codecov coverage.

Scope
=====

This adds codecov coverage report to the repo.

For empirical we don't look at full coverage, but this needs to be done as the script which triggers the coverage (nose_runner.py) is part of this repo.


Changes
=======

buildmaster was updated to set the CODECOV_TOKEN env so that coverage is enabled.

udpate brink and enable coverage publishing.

as a drive by add github PR template and paver.sh


How to test
=========

reviewers: @alibotean 

check that changes are ok.


I tried to fix the `codecov/changes` status, but it looks broken see https://github.com/codecov/support/issues/164 or https://github.com/codecov/support/issues/162.
